west.yml

```
@@ -102,7 +102,7 @@ manifest:
     - name: nrfxlib
       repo-path: sdk-nrfxlib
       path: nrfxlib
-      revision: 2198a0984daeece7adc4b2bea2f1863463e16285
+      revision: revision: pull/328/head
```

Too many revisions here:
```suggestion
      revision: pull/328/head 
```